Code/Tools/AssetProcessor/native/tests/assetmanager/AssetProcessorManagerTest.cpp

```diff
@@ -176,6 +180,20 @@ void AssetProcessorManagerTest::SetUp()
 
     AssetProcessorTest::SetUp();
 
+    AZ::AllocatorInstance<AZ::PoolAllocator>::Create();
+    AZ::AllocatorInstance<AZ::ThreadPoolAllocator>::Create();
+
+    AZ::JobManagerDesc jobDesc;
+    AZ::JobManagerThreadDesc threadDesc;
+    for (int i = 0; i < 4; ++i)
```

Comment:
Same feedback here as the last point. Can you either comment this or make it a const, so readers will know the significance of this value?